app/cloud/server/methods.js

```diff
@@ -46,6 +73,18 @@ Meteor.methods({
 
 		return connectWorkspace(token);
 	},
+	'cloud:disconnectWorkspace'() {
+		if (!Meteor.userId()) {
+			throw new Meteor.Error('error-invalid-user', 'Invalid user', { method: 'cloud:connectServer' });
+		}
+
+		if (!hasPermission(Meteor.userId(), 'manage-cloud')) {
+			throw new Meteor.Error('error-not-authorized', 'Not authorized', { method: 'cloud:connectServer' });
+		}
+
+		return disconnectWorkspace();
+	},
+	// Currently unused but will link local account to Rocket.Chat Cloud account.
 	'cloud:getOAuthAuthorizationUrl'() {
```

Comment:
Is this `cloud:getOAuthAuthorizationUrl ` still necessary?

Reply:
Not being used currently.  It’s possible we might use it in the future?